index.html

```diff
@@ -1,298 +1,341 @@
-                        <a href="#" class="twitter"><img src="../setup-and-mobile-version-skeleton/images/Twiter.svg" alt=""></a>
-                    </li>
-                    <li>
-                        <a href="#" class="linkedin"><img src="../setup-and-mobile-version-skeleton/images/Linkedin.svg" alt=""></a>
+                        <input id="name" type="text" placeholder="Please enter your name" name="user_name" value=""
+                            maxlength="30" required>
                     </li>
+
                     <li>
-                        <a href="#" class="M"><img src="../setup-and-mobile-version-skeleton/images/M.svg" alt=""></a>
+                        <input type="email" name="user_email" required placeholder="Please enter your email">
+
                     </li>
                     <li>
-                        <a href="#" class="C"><img src="../setup-and-mobile-version-skeleton/images/C.svg" alt=""></a>
+                        <textarea id="message" name="user_message" placeholder="Write your message here"
+                            maxlength="500" required></textarea>
                     </li>
                     <li>
-                        <a href="#" class="2f"><img src="../setup-and-mobile-version-skeleton/images/2f.svg" alt=""></a>
+                        <button type="submit">Get In Touch</button>
                     </li>
                 </ul>
-            </div>
-            
-      </div>
-      <div class="view-resume">
-        <div><a class="button-project" href="#">Get my resume</a></div>
-    </div>
-      </div>
-      
-      <div class="lang-select">
-          <div class="language-label">Languages</div>
-          <div class="lang-arrow-button"><a href="#"><img alt=" " src="../setup-and-mobile-version-skeleton/images/icons8-expand-arrow-30.png"></a></div>
-      </div>
-
-      <div class="lang-box-container">
-          
-        <div class="lang-box">
-              <div class="lang-logo-js"></div>
-              <div class="lang-label">javascript</div>
-          </div>
-
-          <div class="lang-box">
-            <div class="lang-logo-html"></div>
-            <div class="lang-label">HTML</div>
-          </div>
-
-          <div class="lang-box">
-            <div class="lang-logo-css"></div>
-            <div class="lang-label">CSS</div>
-          </div>
-
-      </div>
-      <div class="framework-select">
-          <div class="framework-label">Framework</div>
-          <div class="frameW-arrow-button"><a href="#"><img alt=" " src="../setup-and-mobile-version-skeleton/images/icons8-right-arrow-32.png"></a></div>
-      </div>
-
-      <div class="skills-select">
-        <div class="skills-label">Skills</div>
-        <div class="skills-arrow-button"><a href="#"><img alt=" " src="../setup-and-mobile-version-skeleton/images/icons8-right-arrow-32.png"></a></div>
-    </div>
+            </form>
+        </div>
+    </section>
+    <!-- <footer></footer>-->
+</body>
```

Comment:
- [ ] Please do not commit old pieces of code (line 338) as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.